tests-e2e/cypress/integration/backstage/backstage_spec.js

```diff
@@ -65,4 +67,55 @@ describe('backstage', () => {
         // * Verify that incidents are shown
         cy.findByTestId('titleIncident').should('be.visible').contains('Incidents');
     });
+    it('opens playbook creation page with New Playbook button', () => {
```

Comment:
nit: missing newline separating tests
```suggestion

     it('opens playbook creation page with New Playbook button', () => { 
```

Reply:
Related, this test feels like it belongs in the `playbook_list_spec.js`? I had imagined the vanilla `backstage_spec.js` was more about the shared elements, like the navigation header.

Reply:
I agree that it does not quite fit in the `backstage_spec`. Maybe I'll move it to a `playbook_creation_spec`.